apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokensList/PortfolioDAppsTokenList/PortfolioDAppsTokenList.tsx

```diff
@@ -45,6 +47,22 @@ export const PortfolioDAppsTokenList = () => {
     return listLiquidityPool
   }, [isSearching, search, liquidityPools])
 
+  React.useEffect(() => {
+    let timeout: NodeJS.Timeout | undefined
```

Comment:
I've had some issues with this in the past, because the type of timeouts changes depending on the environment. Clearly `NodeJS.Timeout` doesn't sound right in a React Native context. It doesn't complain here because we have node types in the tsconfig, which we share between the app and the tests (they could be separate configs to be more accurate).

I belive the best way to type timeouts is
```
ReturnType<typeof setTimeout>
```